src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    public Excavator(String machine) {
+        super(machine);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println(machine + " started its work.");

```

Comment:
The work message is not informative enough. It should clearly indicate that an `Excavator` is working. For example: "Excavator started its work."

Reply:
C:\Users\48572\.jdks\openjdk-22.0.2\bin\java.exe "-javaagent:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2024.1.4\lib\idea_rt.jar=53355:C:\Program Files\JetBrains\IntelliJ IDEA Community Edition 2024.1.4\bin" -Dfile.encoding=UTF-8 -Dsun.stdout.encoding=UTF-8 -Dsun.stderr.encoding=UTF-8 -classpath C:\Users\48572\IdeaProjects\jv-oop\target\classes core.basesyntax.MainApp
Truck started its work.
Truck stopped working. 
Bulldozer started its work.
Bulldozer stopped working.
 Excavator started its work.
 Excavator stopped working.
[Truck, Bulldozer,  Excavator]